app/src/main/java/com/github/se/polyfit/model/recipe/Recipe.kt

```diff
@@ -0,0 +1,63 @@
+package com.github.se.polyfit.model.recipe
+
+import android.util.Log
+import java.net.URL
+
+data class Recipe(
+    val id: Long,
+    val title: String,
+    val imageUrl: URL,
+    val usedIngredients: Long,
+    val missingIngredients: Long,
```

Comment:
This is done to represent the fields of the API as much as closely as possible ([Spoonacular API](https://rapidapi.com/spoonacular/api/recipe-food-nutrition)). Essentially, it includes the ingredients that the user has (`usedIngredients`) and the ingredients that are missing (`missingIngredients`) for a specific recipe. I thought this could be useful for future features.